src/blog/2024/01/unified-namespace-when-not-to-use.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Unified Namespace: When to Use It, and When to Choose Something Else"
+subtitle: Data isn't created equal, some data doesn't fit the UNS 
+description: Data isn't created equal, some data doesn't fit the UNS
+date: 2024-01-31
+authors: ["zeger-jan-van-de-weg"]
+image: blog/2024/01/images/when-not-to-use-the-uns.png
+tags:
+    - posts
+    - flowfuse
+    - unified-namespace
+---
+
+At FlowFuse, we're convinced of the Unified Namespace (UNS) architecture for IoT cases. It's a powerful tool that can make information much more readily available and easy to consume. However, as with any architecture, there are times when it's not the best choice. In this blog post, we'll discuss when to use the UNS and when to consider other options.
+
+<!--more-->
+
+### Latency sensitivity
+
+When automating tasks, performing tasks with requirements on latency aren’t too common in a digital environment. Requesting APIs from another server, normalizing data, and sending it towards another service takes very little time, though it hardly matters if you’re 100ms later than usual. In industrial automation or other cases where physical safety is safeguarded, there’s a requirement for low latency to ensure that data is transmitted and processed quickly enough to maintain real-time control of processes.
```

Comment:
I am having trouble following the first sentence.

"When doing automation, latency isn't a common issue in the digital environment?"

Reply:
```suggestion
When automating strictly digital tasks, there's generally no or very lenient requirements on latency. Requesting APIs from another server, normalizing data, and sending it towards another service takes very little time, though it hardly matters if you’re 100ms later than usual. In industrial automation or other cases where physical safety is safeguarded, there’s a requirement for low latency to ensure that data is transmitted and processed quickly enough to maintain real-time control of processes.
```